### PR TITLE
Out out damm deprecations

### PR DESCRIPTION
**Pull request by rlivsey**

### CODE REVIEW

File: addon/mixins/sortable-item.js [hunk 1]
```diff
@@ -187,14 +187,18 @@ export default Mixin.create({
   */
   didInsertElement() {
     this._super();
-    this._tellGroup('registerItem', this);
+    // scheduled to prevent deprecation warning:
+    // "never change properties on components, services or models during didInsertElement because it causes significant performance degradation"
+    Ember.run.schedule("afterRender", this, "_tellGroup", "registerItem", this);
```
Comment (jgwhite) [inline addon/mixins/sortable-item.js]: You can use `run.schedule`, it’s been plucked off `Ember` further up.

File: addon/mixins/sortable-item.js [hunk 2]
```diff
@@ -187,14 +187,18 @@ export default Mixin.create({
...
   },
 
   /**
     @method willDestroyElement
   */
   willDestroyElement() {
-    this._tellGroup('deregisterItem', this);
+    // scheduled to prevent deprecation warning:
+    // "never change properties on components, services or models during didInsertElement because it causes significant performance degradation"
+    Ember.run.schedule("afterRender", this, "_tellGroup", "deregisterItem", this);
```
Comment (jgwhite) [inline addon/mixins/sortable-item.js]: Same here.

**Comment by jgwhite** [comment]
Thanks!!

**Comment by rlivsey** [comment]
Updated as per comments.

Got to love 1 liners that take 3 hours to track down…

To be fair, that's what I was going to do 3 hours back but I wanted to figure out *why* it was breaking!

**Comment by jgwhite** [comment]
LGTM

**Comment by jgwhite** [comment]
Released as v1.5.2

**Pull request merged**